synapse/logging/opentracing.py

```diff
@@ -738,3 +756,7 @@ class _DummyTagNames(object):
 
 
 tags = _DummyTagNames
+
+
+def get_tags():
```

Comment:
Instead of doing this hack, lets do:

```python
try:
    import opentracing
    tags = opentracing.tags
except ImportError:
    opentracing = None
    tags = _DummyTagNames
```

Or something